### PR TITLE
Fix interop Python test invocation

### LINKED ISSUE #4317: Interop test fails with "tox command not found"
Failing for PR #4306, which tries to bring into master a commit called "Use tox" :)

[`./tools/run_tests/build_python.sh: line 47: tox: command not found`](https://grpc-testing.appspot.com/job/gRPC_interop_pull_requests/654/console).

Stack trace:
```
++ dirname ./tools/run_tests/build_python.sh
+ cd ./tools/run_tests/../..
++ pwd
+ ROOT=/var/local/git/grpc
+ GRPCIO=/var/local/git/grpc/src/python/grpcio
+ export LD_LIBRARY_PATH=/var/local/git/grpc/libs/opt
+ LD_LIBRARY_PATH=/var/local/git/grpc/libs/opt
+ export DYLD_LIBRARY_PATH=/var/local/git/grpc/libs/opt
+ DYLD_LIBRARY_PATH=/var/local/git/grpc/libs/opt
+ export PATH=/var/local/git/grpc/bins/opt:/var/local/git/grpc/bins/opt/protobuf:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+ PATH=/var/local/git/grpc/bins/opt:/var/local/git/grpc/bins/opt/protobuf:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+ export 'CFLAGS=-I/var/local/git/grpc/include -std=c89'
+ CFLAGS='-I/var/local/git/grpc/include -std=c89'
+ export LDFLAGS=-L/var/local/git/grpc/libs/opt
+ LDFLAGS=-L/var/local/git/grpc/libs/opt
+ export GRPC_PYTHON_BUILD_WITH_CYTHON=1
+ GRPC_PYTHON_BUILD_WITH_CYTHON=1
+ export GRPC_PYTHON_ENABLE_CYTHON_TRACING=1
+ GRPC_PYTHON_ENABLE_CYTHON_TRACING=1
+ cd /var/local/git/grpc/src/python/grpcio
+ tox --notest
./tools/run_tests/build_python.sh: line 47: tox: command not found
+ EXITCODE=127
+ docker rm -f build_grpc_interop_python_7281ba4c-49c1-4312-808a-31818efe7cef
build_grpc_interop_python_7281ba4c-49c1-4312-808a-31818efe7cef
+ exit 127

FAILED: build_docker_python [ret=127, pid=10854]
```

### PR DESCRIPTION
**Pull request by soltanmm**
Fixes #4317.

Fixes a fix that was missing a fix in said fix. The ideal fix would fix a `tox` command fixating on the interop server and client.

tl;dr: I'm a moose.

cc @nathanielmanistaatgoogle
cc @jcanizales

### CODE REVIEW

File: src/python/grpcio/tox.ini [hunk 1]
```diff
@@ -15,8 +15,5 @@ commands =
     coverage html --include='grpc/*' --omit='grpc/framework/alpha/*','grpc/early_adopter/*','grpc/framework/base/*','grpc/framework/face/*','grpc/_adapter/fore.py','grpc/_adapter/rear.py'
     coverage report --include='grpc/*' --omit='grpc/framework/alpha/*','grpc/early_adopter/*','grpc/framework/base/*','grpc/framework/face/*','grpc/_adapter/fore.py','grpc/_adapter/rear.py'
 deps =
-    cython
-    coverage
-    oauth2client
-    protobuf
+    -rrequirements.txt
```
Comment (nathanielmanistaatgoogle) [inline src/python/grpcio/tox.ini]: The absence of space between `-r` and `requirements.txt` on this line is a legitimate thing and not a typo, right?

Comment (soltanmm) [inline src/python/grpcio/tox.ini]: Yep.

**Comment by nathanielmanistaatgoogle** [comment]
Change content looks fine; waiting on test results.

**Comment by jtattermusch** [comment]
LGTM once interop tests pass. Currently it looks like it's failing because it can't get over PHP.

**Comment by jtattermusch** [comment]
Ruby interop test failure is unrelated.
